From afb83381d24e0b0a46c748ce393296aef58b92b5 Mon Sep 17 00:00:00 2001
From: Ilia Lin <ilialin@codeaurora.org>
Date: Tue, 28 Oct 2014 16:02:56 +0200
Subject: [PATCH 11/11] ARM: dts: msm: Enable APQ8094 DB qpnp gpios

Enable the qpnp gpios disabled globally on the
Dragonboard APQ8094.

Change-Id: Id971382b853e0575283a5f607fae4d81dc6b7e87
Signed-off-by: Ilia Lin <ilialin@codeaurora.org>
---
 arch/arm/boot/dts/qcom/apq8094-dragonboard.dtsi | 30 +++++++++++++++++++++++++
 1 file changed, 30 insertions(+)

diff --git a/arch/arm/boot/dts/qcom/apq8094-dragonboard.dtsi b/arch/arm/boot/dts/qcom/apq8094-dragonboard.dtsi
index 894b331..1bc9e1e 100644
--- a/arch/arm/boot/dts/qcom/apq8094-dragonboard.dtsi
+++ b/arch/arm/boot/dts/qcom/apq8094-dragonboard.dtsi
@@ -606,6 +606,7 @@
 		qcom,vin-sel = <2>;	/* 1.8v */
 		qcom,src-sel = <0>;	/* Constant */
 		qcom,master-en = <1>;	/* Enable GPIO */
+		status = "okay";
 	};
 
 	gpio@c200 { /* GPIO 3 - KEY_VOLP_N */
@@ -614,6 +615,7 @@
 		qcom,vin-sel = <2>;
 		qcom,src-sel = <0>;
 		qcom,master-en = <1>;
+		status = "okay";
 	};
 
 	gpio@c300 { /* GPIO 4 - EDU_GPIO2 */
@@ -621,6 +623,7 @@
 		qcom,pull = <1>;
 		qcom,vin-sel = <2>;
 		qcom,src-sel = <0>;
+		status = "okay";
 	};
 
 	gpio@c400 { /* GPIO 5 - BL1_PWM */
@@ -628,6 +631,7 @@
 		qcom,pull = <1>;	/* PULL DOWN */
 		qcom,vin-sel = <2>;	/* 1.8 */
 		qcom,src-sel = <4>;	/*  */
+		status = "okay";
 	};
 
 	gpio@c500 { /* GPIO 6 - EXT_CHARGER_INT */
@@ -635,6 +639,7 @@
 		qcom,pull = <1>;
 		qcom,vin-sel = <2>;
 		qcom,src-sel = <0>;
+		status = "okay";
 	};
 
 	gpio@c600 { /* GPIO 7 - NFC_PWR_REQ */
@@ -643,6 +648,7 @@
 		qcom,vin-sel = <2>;	/* 1.8 */
 		qcom,src-sel = <0>;	/* CONSTANT */
 		qcom,master-en = <1>;	/* ENABLE GPIO */
+		status = "okay";
 	};
 
 	gpio@c700 { /* GPIO 8 - SD_CARD_DET_N */
@@ -651,6 +657,7 @@
 		qcom,vin-sel = <2>;
 		qcom,src-sel = <0>;
 		qcom,master-en = <1>;
+		status = "okay";
 	};
 
 	gpio@c900 { /* GPIO 10 - NFC_CLK_REQ */
@@ -659,6 +666,7 @@
 		qcom,vin-sel = <2>;	/* 1.8 */
 		qcom,src-sel = <2>;	/* QPNP_PIN_SEL_FUNC_1 */
 		qcom,master-en = <1>;	/* ENABLE GPIO */
+		status = "okay";
 	};
 
 	gpio@ca00 { /* GPIO 11 - FP_LINEOUT_PLUG_DET_N */
@@ -666,6 +674,7 @@
 		qcom,pull = <5>; /* No Pull */
 		qcom,vin-sel = <2>; /* 1.8 */
 		qcom,src-sel = <0>; /* CONSTANT */
+		status = "okay";
 	};
 
 	gpio@cc00 { /* GPIO 13 - RP_LINEOUT_PLUG_DETCT */
@@ -673,6 +682,7 @@
 		qcom,pull = <5>;	/* No Pull */
 		qcom,vin-sel = <2>;	/* 1.8 */
 		qcom,src-sel = <0>;	/* CONSTANT */
+		status = "okay";
 	};
 
 	gpio@cd00 { /* GPIO 14 - BL1_EN */
@@ -681,6 +691,7 @@
 		qcom,vin-sel = <2>;
 		qcom,src-sel = <0>;
 		qcom,master-en = <1>;
+		status = "okay";
 	};
 
 	gpio@ce00 { /* GPIO 15 - DIVCLK1 */
@@ -691,6 +702,7 @@
 		qcom,out-strength = <1>;
 		qcom,src-sel = <2>;
 		qcom,master-en = <1>;
+		status = "okay";
 	};
 
 	gpio@cf00 { /* GPIO 16 - DIVCLK2 */
@@ -701,6 +713,7 @@
 		qcom,out-strength = <1>;
 		qcom,src-sel = <2>;
 		qcom,master-en = <1>;
+		status = "okay";
 	};
 
 	gpio@d100 { /* GPIO 18 - DIVCLK4 */
@@ -711,6 +724,7 @@
 		qcom,out-strength = <1>;
 		qcom,src-sel = <2>;
 		qcom,master-en = <1>;
+		status = "okay";
 	};
 
 	gpio@d200 { /* GPIO 19 - BT_EN */
@@ -719,9 +733,11 @@
 		qcom,vin-sel = <2>;      /* VIN2 */
 		qcom,src-sel = <0>;      /* GPIO */
 		qcom,master-en = <1>;    /* Enable GPIO */
+		status = "okay";
 	};
 
 	gpio@d300 { /* GPIO 20 - PMI_SPON */
+		status = "okay";
 	};
 };
 
@@ -731,6 +747,7 @@
 		qcom,pull = <1>;	/* PULL DOWN */
 		qcom,vin-sel = <2>;	/* 1.8 */
 		qcom,src-sel = <0>;	/* CONSTANT */
+		status = "okay";
 	};
 
 	mpp@a300 { /* MPP 4 - EDU_MPP4 */
@@ -738,6 +755,7 @@
 		qcom,pull = <1>;	/* PULL DOWN */
 		qcom,vin-sel = <2>;	/* 1.8 */
 		qcom,src-sel = <0>;	/* CONSTANT */
+		status = "okay";
 	};
 
 	mpp@a400 { /* MPP 5 - EDU_MPP2 */
@@ -745,6 +763,7 @@
 		qcom,pull = <1>;	/* PULL DOWN */
 		qcom,vin-sel = <2>;	/* 1.8 */
 		qcom,src-sel = <0>;	/* CONSTANT */
+		status = "okay";
 	};
 
 	mpp@a500 { /* MPP 6 - EDU_MPP3 */
@@ -752,6 +771,7 @@
 		qcom,pull = <1>;	/* PULL DOWN */
 		qcom,vin-sel = <2>;	/* 1.8 */
 		qcom,src-sel = <0>;	/* CONSTANT */
+		status = "okay";
 	};
 
 	mpp@a600 { /* MPP 7 - DBU1_EN */
@@ -760,6 +780,7 @@
 		qcom,vin-sel = <2>;	/* 1.8 */
 		qcom,src-sel = <0>;	/* CONSTANT */
 		qcom,master-en = <1>;
+		status = "okay";
 	};
 
 	mpp@a700 { /* MPP 8 - EXP_LED_SINK */
@@ -768,6 +789,7 @@
 		qcom,vin-sel = <2>;	/* 1.8 */
 		qcom,src-sel = <0>;	/* CONSTANT */
 		qcom,master-en = <1>;
+		status = "okay";
 	};
 };
 
@@ -778,6 +800,7 @@
 		qcom,vin-sel = <2>;
 		qcom,src-sel = <0>;
 		qcom,master-en = <1>;
+		status = "okay";
 	};
 
 	gpio@c200 { /* GPIO 3 - EDU_GPIO1 */
@@ -785,6 +808,7 @@
 		qcom,pull = <1>;
 		qcom,vin-sel = <2>;
 		qcom,src-sel = <0>;
+		status = "okay";
 	};
 
 
@@ -794,6 +818,7 @@
 		qcom,vin-sel = <2>;/*  1.8 */
 		qcom,src-sel = <0>;
 		qcom,master-en = <1>;	/* ENABLE GPIO */
+		status = "okay";
 	};
 
 	gpio@c600 { /* GPIO 7 - EDU_GPIO3 */
@@ -801,6 +826,7 @@
 		qcom,pull = <1>;
 		qcom,vin-sel = <2>;
 		qcom,src-sel = <0>;
+		status = "okay";
 	};
 };
 
@@ -811,12 +837,15 @@
 		qcom,src-sel = <7>;	/* DTEST4 */
 		qcom,vin-sel = <2>;	/* 1.8 */
 		qcom,master-en = <1>;	/* Enable MPP */
+		status = "okay";
 	};
 
 	mpp@a100 { /* MPP 2 - FLASH_STROBE (DNI) */
+		status = "okay";
 	};
 
 	mpp@a200 { /* MPP 3 - PMI_SPON */
+		status = "okay";
 	};
 
 	mpp@a300 { /* MPP 4 - EXT_FET_WLED_PWR_EN */
@@ -824,6 +853,7 @@
 		qcom,src-sel = <0>;	/* FUNC GPIO */
 		qcom,vin-sel = <0>;	/* VPH_PWR */
 		qcom,master-en = <1>;	/* Enable MPP */
+		status = "okay";
 	};
 };
 
-- 
1.8.2.1

